src/main.rs

```diff
@@ -132,73 +154,22 @@ extern "C" fn sample_main() {
         // Wait for either a specific button push to exit the app
         // or an APDU command
         if let Event::Command(ins) = ui_menu_main(&mut comm) {
```

Comment:
For clarity shouldn't we write 

```suggestion
        if let Event::Command(header) = ui_menu_main(&mut comm) {
```
?